recipes/cuda-compat/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "cuda-compat" %}
+{% set version = "525.60.13" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set extension = "tar.xz" %}  # [not win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/nvidia_driver/{{ platform }}/nvidia_driver-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: a42691822f6e87060da5682a8e0662c86d1b20fbf31d12cb2969b819995e4138  # [linux64]
+  sha256: 4ad08f8431195580349f4984fae2d93697d2f5572837c1e7ae59c37feab291c8  # [ppc64le]
+  sha256: 1708ae3ba857559cbfc4e6c06267ed6126eff2c68fa5321a95906f50e984148c  # [aarch64]
+
+build:
+  number: 0
+  skip: true  # [osx]
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
```

https://github.com/conda-forge/staged-recipes/pull/21952#issuecomment-1468646082
We might need CUDA 12 drivers for this package to be useful.
```suggestion
    - {{ compiler("cxx") }}
  run_constrained:
    - __cuda >=12
```

---

Do we need a more specific pinning for driver version? Or is a lower bound like this sufficient?